### ISSUE TITLE
[Feature] Add option to not display sccache stats in workflow summary

### ISSUE THREAD

**Issue by lcheylus** [Mozilla-Actions/sccache-action#125]
With PR #113, there is now an annotation and a summary table with sccache stats for each job at the end of a workflow. sccache stats are also available in a "Post Run" step for each job.

For some jobs, I would like to have an option for this action to **NOT add "sccache stats" in annotation and summary table**. For example, add option `stats` (`true` by default):
```yaml
uses: mozilla-actions/sccache-action@v0.0.5
with:
  version: "v0.7.4"
  stats: false
```

@orf As author of this PR, please could you add this option ?

**Arguments for my request:**
- I'm using the [vmactions/freebsd-vm](https://github.com/vmactions/freebsd-vm) action to run some jobs/workflow on FreeBSD OS => FreeBSD VM run on an Ubuntu GH runner.
- In this FreeBSD VM, I'm using `sccache` action: install `sccache` FreeBSD package and set some env. vars for the action => `sccache` is correctly used in my job (build of a Rust project via `cargo build`).
- At the end of the FreeBSD job, "**sccache stats" are always void** because the command `sccache --show-stats` run on the Ubuntu runner, not on the FreeBSD VM where `sccache` is used.
- In conclusion, sccache stats displayed in annotation/summary table are useless in this case.
- I have a specific step/script in my FreeBSD workflow to get "sccache stats" in a text file and displays it.

**Comment by whisperity** [comment]
I believe this was fixed by #162.

**Comment by lcheylus** [comment]
Thanks, tests with option `disable_annotations` => no stats report for sccache at the end of my workflow :)

Issue solved

**Comment by sylvestre** [comment]
magnifique! thanks for closing the loop :)
